com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/UnityBridge.kt

```diff
@@ -57,6 +58,21 @@ class UnityBridge {
             }
         }
 
+        @JvmStatic
+        fun loadBannerAd(listener: ChartboostMediationBannerViewListener): BannerAdWrapper{
+            val size = HeliumBannerAd.HeliumBannerSize.STANDARD;
+            val bannerView = HeliumBannerAd(UnityPlayer.currentActivity,"", size, null);
+            val bannerAdWrapper = BannerAdWrapper.wrap(bannerView);
+            bannerAdWrapper.setListener(listener);
+//            AdStore.trackBannerAd(bannerAdWrapper);
```

Comment:
Is this commented code intentional?

Reply:
@cb-jpadilla @kushG I don't think the AdStore is being used at all for the current Adaptive Banner implementation, is it?